python/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -58,9 +58,9 @@ def getReplicas(lfns):
     output(dirac.getReplicas(lfns, active=True, preferDisk = True))
 
 
-def getAccessURL(lfn,SE):
-    ''' Return the access URL for the given LFN and storage element '''
-    output(dirac.getAccessURL(lfn,SE))
+def getAccessURL(lfn, SE, protocol):
+    ''' Return the access URL for the given LFN, storage element and protocol '''
```

Comment:
Could you add appropriate docs for this function as well? As per the Google python style guide:

http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

We're trying to sort this out across the code base as we go 😄